src/sound/backend/i_sound.h

```diff
@@ -37,6 +37,7 @@
 
 #include "doomtype.h"
 #include "i_soundinternal.h"
+#include "s_sound.h"
```

Comment:
You don’t need to include header file, forward declaration of `FSoundChan` is enough.

Reply:
Actually I did. It wouldn't compile otherwise.

Reply:
Or if I'm mistaken about what you mean... Can you give me an example?

Reply:
Done.